packages/babel-preset-default/index.js

```diff
@@ -1,36 +1,75 @@
 module.exports = function( api ) {
+	let wpBuildOpts = {};
 	const isTestEnv = api.env() === 'test';
+	const isWPBuild = ( name ) => [ 'WP_BUILD_MAIN', 'WP_BUILD_MODULE' ].some(
+		( buildName ) => name === buildName
+	);
+
+	// Because we serve a different preset depending
+	// on the caller options, we need to tell the cache
+	// when it needs updating. Otherwise, it won't be
+	// recalculated for different builds.
+	api.cache.using( () => isWPBuild( wpBuildOpts.name ) );
+
+	api.caller( ( caller ) => {
+		if ( caller && isWPBuild( caller.name ) ) {
+			wpBuildOpts = { ...caller };
+		}
+	} );
+
+	const getPresetEnv = () => {
+		const opts = {};
+
+		if ( isTestEnv ) {
+			opts.useBuiltIns = 'usage';
+		} else {
+			opts.modules = false;
+			opts.targets = {
+				browsers: require( '@wordpress/browserslist-config' ),
+			};
+		}
+
+		if ( isWPBuild( wpBuildOpts.name ) ) {
+			opts.modules = wpBuildOpts.modules;
+		}
+
+		return [ require.resolve( '@babel/preset-env' ), opts ];
+	};
+
+	const maybeGetPluginTransformRuntime = () => {
+		if ( isTestEnv ) {
+			return undefined;
+		}
+
+		const opts = {
+			helpers: true,
+			useESModules: false,
```

Comment:
How about:

```suggestion
			useESModules: wpBuildOpts.name === 'WP_BUILD_MODULE' ? wpBuildOpts.useESModules : false,
```

Reply:
I prefer to leave it as it is so it's similar to what we have in `getPresetEnv`.